src/Microsoft.SymbolStore/SymbolStores/HttpSymbolStore.cs

```diff
@@ -15,6 +15,12 @@
 
 namespace Microsoft.SymbolStore.SymbolStores
 {
+    public class SymbolAuthHeader
+    {
+        public string Scheme { get; set; }
+        public string Parameter { get; set; }
```

Comment:
Should we make this an immutable class that throws in the constructor if either of these is null?

Reply:
Actually, I wonder if you want to get rid of this type and instead just pass these as arguments. The reason I suggest this is that you now have two constructors with the same number of arguments. So if someone does `new HttpSymbolStore(tracer, backingStore, symbolServerUri, null)` it is ambiguous which constructor to call. Where as if you get rid of this type and just add two arguments to the constructor it is no longer ambiguous.

Reply:
I like the idea of removing the SymbolAuthHeader class and passing the members as parameters to the new constructor.